### ISSUE TITLE
Not working yet

### ISSUE THREAD

**Issue by raphaelnn** [joax/thunderbird-mentions#2]
Hi,

in my scenario, I am using TB 78.5.1 64-bit on Windows 10.
My email is IMAP and my address book is through TBSync, but local address book doesn't work either.
When typing '@', nothing happens, as well as when I click "@ Add mentions".

**Comment by joax** [comment]
Hello Raphael, the contact search will activate on the body of the composing email when you type Space + @. Otherwise, it may be interrupting your writing of an email. Can you try that and see if it works?

The button "Add Mentions" only works after adding the mentions on the body. Its function is to add the mentions on the email to the CC field.

**Comment by raphaelnn** [comment]
Hi Joaqin.
I've tried exactly that, with no success.
When I type Space + @ and type the contact, there is no attempt to auto-complete it.

**Comment by joax** [comment]
Let me open the same setup and test it. One last thing, can you try once again opening a new composer window and clicking on the body of the email (instead of using tab key) before doing the Space + @?

**Comment by raphaelnn** [comment]
Did everything with the mouse, not with keyboard shortcuts, and the result was the same.

**Comment by joax** [comment]
Ok, I am setting it up to test. Will get back to you with the result. Thank you so much for your help.

**Comment by joax** [comment]
So I was able to setup the same configuration. Do you see the "Search contacts..." overlay when you type the Space + @ at the composer window? or just nothing happens?

**Comment by raphaelnn** [comment]
Nothing happens

![image](https://user-images.githubusercontent.com/11463840/102423649-46ef9100-3fe8-11eb-97d3-4ed40b6e18bf.png)

**Comment by joax** [comment]
Something must be going on. I was able to see it working on my windows setup with IMAP + TbSync (I use the same setup). Are you familiar with the Developer tools of your Thunderbird? 

Menu > Tools > Developer Tools > Developer Toolbox. 

I would love to check what is going on on your side to be able to fix it. 

![Screenshot from 2020-12-16 17-13-44](https://user-images.githubusercontent.com/265957/102425454-2f051680-3fc2-11eb-8e17-aee5823f5f56.png)

**Comment by raphaelnn** [comment]
I am trying to find something. Do you have any addon that mess with keyboard shortcuts installed?

**Comment by raphaelnn** [comment]
This is what I got

![image](https://user-images.githubusercontent.com/11463840/102434052-bb2f3200-3ff2-11eb-9f56-3df1da24d8f0.png)

**Comment by raphaelnn** [comment]
This is what I got

![image](https://user-images.githubusercontent.com/11463840/102434082-ce420200-3ff2-11eb-9b61-5b278ff04f32.png)

**Comment by joax** [comment]
It doesn't. It just binds to the keydown listener on the composeActions. It shouldn't. 
Can you give me the add-ons you have installed so I can reproduce the same environment? 
There might be another add-on overlapping with the event listener.

**Comment by raphaelnn** [comment]
![image](https://user-images.githubusercontent.com/11463840/102437534-6e9b2500-3ff9-11eb-8312-5386479d4c5e.png)

**Comment by joax** [comment]
Tested the new release in Windows with all your plugins. I found a [bug](https://thunderbird.topicbox.com/groups/addons/Tb109bd72d5d456d7/composescripts-window-listeners-not-working-if-not-clicking-on-the-compose-body) on Thunderbird that impacts the focus on the composer body. Probably the issue you are experiencing. Try the new release and see if it helps. Otherwise, I will pursue the TB bug tracker. You can also download the Beta version of Thunderbird (84) and try it there, but this would be at your own risk.

**Comment by raphaelnn** [comment]
I am gonna try it and let you know.
Thanks.

**Comment by raphaelnn** [comment]
Yeah, bad idea. TBSync is not supported, so there is no address book to consult.
Even though, I tried the plugin and there is no contact search box. Maybe we will need to wait the bug to be solved.
Thank you very much for your time and effort to sort this out.

**Comment by raphaelnn** [comment]
Hi Joaquin. I just found the problem.
My keyboard is in spanish-latin american.
My keystroke to summon @ is altgr+q
When I change the layout to english, the keystroke for @ is shift+2, and that works.
So, my guess is, the plugin is awaiting for that specific keystroke and not the @ char.

Hope to help solve this!

Thanks!!

**Comment by joax** [comment]
Thank you so much for the details. I will change this on the plugin so it's
locale agnostic.

**Comment by raphaelnn** [comment]
One last thing I just noted.
Options > Composing > Addressing
If I leave the 'All Address Books' selected in "Default startup directory in the address book window", I can not summon the plugin, no matter the layout I choose.

Thanks!!

**Comment by joax** [comment]
Issue fixed (Spanish and other Keyboards supported). I will open a new issue with the adressbook selection matter.

**Comment by raphaelnn** [comment]
Hi Joaquin. I've just updated the plugin, and now it doesn't work at all, no matter which layout I select.

**Comment by joax** [comment]
Hello, I have rolled back the changes, the version 0.6.4 should work again as intended. Sorry about that!

**Comment by revad** [comment]
I suspect my experience may be related to this issue.
I'm using 78.10.0 (64-bit)  on SuSE Linux. Addon v0.6.4
This is a UK keyboard (@ is 2 keys to the right of L, shifted) and my locale is en-GB. 
If I type
Test @
The @ does not display. If I type @ again it does display (once) - but if I continue typing a name nothing happens.

So, to an inattentive potential user, the addon does nothing.

Only "Mention JS loaded." on the console. 
It does work on TB 89.0b4.

**Comment by revad** [comment]
The problem in my previous comment only happens on the first line in the plain text composer.
On the first line if I just type '@' it works, but '@Test @' or 'TestTAB@'  does not.
On the second line 'Test @' or 'TestTAB@' works.
(I realised that TB 89.0b and TB90.0a were both using the html composer.)

**Comment by raphaelnn** [comment]
Hi Joaquin.
As stated before, the issue remains when using a keystroke different of shift+2
Just tested.
